src/libs/protocols.liq

```diff
@@ -346,14 +324,11 @@ def ffmpeg_protocol(~rlog,~maxtime,arg) =
     list.fold(f,[],lines)
   end
 
-  def replaygain_filter(file) =
+  def replaygain_filter(fname) =
     if replaygain then
-      # The extraction program
-      extract_replaygain = settings.protocol.replaygain.path()
-      delay = maxtime - time()
-      ret = exec_replaygain(delay=delay,extract_replaygain=extract_replaygain,file)
```

Comment:
Does `exec_replaygain` also need to be removed?

Reply:
`extract_replaygain` is removed yes!